app/main.py

```diff
@@ -1,3 +1,56 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance | int") -> "Distance":
```

Comment:
why do u need to use quotes here?
the same is for other annoatations

Reply:
Done